Malaria Prevention App/ViewControllers/Info Hub and Games/Games Lists/Games/Rapid Fire/RapidFireGame.swift

```diff
@@ -6,22 +6,49 @@ class RapidFireGame {
   
   var entries: [RapidFireGameEntry] = []
   var _maximumScore: Int = 0
-
+  
   init() {
-    let entry1 = RapidFireGameEntry(question: "Melfoquine should be taken:",
-                                    answers: ["Daily", "Weekly", "Monthly"],
+    
+    
+    let firstQuestionTitle = NSLocalizedString("Question 1", comment: "")
+    let firstQuestionsAnswers = [NSLocalizedString("Daily", comment: ""),
+                                 NSLocalizedString("Weekly", comment: ""),
+                                 NSLocalizedString("Monthly", comment: "")]
+    
+    let entry1 = RapidFireGameEntry(question: firstQuestionTitle,
+                                    answers: firstQuestionsAnswers,
                                     correctAnswer: 1)
-    let entry2 = RapidFireGameEntry(question: "Malaria is caused by:",
-                                    answers: ["Virus", "Bacteria", "Protozoa"],
+    
+    let secondQuestionTitle = NSLocalizedString("Question 2", comment: "")
+    let secondQuestionsAnswers = [NSLocalizedString("Virus", comment: ""),
+                                  NSLocalizedString("Bacteria", comment: ""),
+                                  NSLocalizedString("Protozoa", comment: "")]
+    
+    let entry2 = RapidFireGameEntry(question: secondQuestionTitle,
+                                    answers: secondQuestionsAnswers,
                                     correctAnswer: 2)
-    let entry3 = RapidFireGameEntry(question: "Doxycycline should be taken: ",
-                                    answers: ["Daily", "Weekly", "Monthly"],
+    
+    let thirdQuestionTitle = NSLocalizedString("Question 3", comment: "")
+    let thirdQuestionsAnswers = firstQuestionsAnswers
+    
+    let entry3 = RapidFireGameEntry(question: thirdQuestionTitle,
+                                    answers: thirdQuestionsAnswers,
                                     correctAnswer: 0)
-    let entry4 = RapidFireGameEntry(question: "Malaria is transmitted through ... mosquito.",
-                                    answers: ["Female Aedes", "Female Anopheles", "Male Aedes"],
+    
+    let fourthQuestionTitle = NSLocalizedString("Question 4", comment: "")
```

Comment:
You can create a global function `localize(id: string, comment: string = "")` to reduce the boiler plate code (and simplify the reading).